dCommon/GeneralUtils.cpp

```diff
@@ -194,6 +196,13 @@ std::string GeneralUtils::UTF16ToWTF8(const std::u16string_view string, const si
 
 	return ret;
 }
+std::string GeneralUtils::Latin1ToWTF8(const std::u8string_view string, const size_t size) {
+	return ToWTF8(string, size);
```

Comment:
This is way too complicated... what I meant is just call `PushUtf8CodePoint` on all bytes, will get you a latin1 decoder. It needs none of the surrgate handling.

Reply:
#1695